gettsim/data/param.yaml

```
@@ -8782,7 +8797,7 @@ ust_high:
     1993-01-01:
       value: .15
       note: .
-    1999-01-01:
+    1999-04-01:
```

```suggestion
    1998-04-01:
```

There was indeed a big amendment of the UStG in 1999, see https://offenegesetze.de/veroeffentlichung/bgbl1/1999/30?q=umsatzsatzsteuer%20sechzehn#page=2 with the new law in force from 1999-04-01.

But the change from .15 to .16 already happened a year before on 1998-04-01, see https://offenegesetze.de/veroeffentlichung/bgbl1/1997/86?q=Umsatzsteuer#page=30 or  https://www.welt.de/print-welt/article645301/Mehrwertsteuer-steigt-zum-1-April-auf-16-Prozent.html.

---

right that was a typo. 

> I would appreciate if the changes of the StaFamG are addressed in a separate PR as they are relatively complex.

I'm actually doing it the other way round now and do only StaFamG stuff here :)

---

> I'm actually doing it the other way round now and do only StaFamG stuff here :)

Pls update the title then.